pkgs/applications/video/f1viewer/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "f1viewer";
+  version = "2.4.0";
+
+  src = fetchFromGitHub {
+    owner = "SoMuchForSubtlety";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "7eXRUG74l9+9nU7EmDvNcHc+2pg5+/amjqtrzT60f94=";
+  };
+
+  vendorSha256 = "4pQ8NT0mh3w7naHEHh2w6Csop0uitlWClZ95VlYaPW0=";
+
+  meta = with lib; {
+    description = "TUI for F1TV.";
+    homepage = "https://github.com/SoMuchForSubtlety/f1viewer";
+    license = licenses.gpl3;
```

Comment:
```suggestion
    license = licenses.gpl3Plus;
```

Reply:
I can't find this information in the repo. I think it should be `gpl3Only`.

Reply:
Line 637:
>  This program is free software: you can redistribute it and/or modify
    it under the terms of the GNU General Public License as published by
    the Free Software Foundation, either version 3 of the License, or
    (at your option) any later version.

Reply:
That's just a recommendation inside the license, but the author of f1viewer didn't put such a specification anywhere.

Reply:
I admittedly am not very sure what a “gpl3Plus” license looks like in a repo, so I went along with the suggestion. But it does look like the repo includes specifically gpl3. When I get home I’ll try to find an example of gpl3Plus to get a better idea of the difference.

Reply:
Developers are advised to include a sentence like the above at the top of every source code file. Only some include the part "or (at your option) any later version."

Reply:
Right, so I see a repo such as https://github.com/robbert-vdh/diopser, packaged as `diopser` in nixpkgs, contains GPL3 license in its repo and has such a disclaimer(? wrong word) in its files. So I understand that that is gpl3Plus. In the case of f1viewer I do not see such a disclaimer in files, and the blurb at line 637 of the license is not actually part of the license itself but instead part of the recommendation to add these disclaimers(? again, there is probably a better word) to the files. So for this package I would be on board with changing the license back to gpl3Only.

Reply:
Exactly.

Reply:
Done. I think that was the last of the review feedback, thank you both. Anything else need to be done before merge? There's apparently an outstanding requested change I don't know how to get rid of, but it was addressed.